man/systemd.exec.xml

```diff
@@ -2304,10 +2304,10 @@ SystemCallErrorNumber=EPERM</programlisting>
         set by the service manager itself (such as <varname>$NOTIFY_SOCKET</varname> and such), or set by a PAM module
         (in case <varname>PAMName=</varname> is used).</para>
 
-        <para>
-        See <citerefentry
-        project='man-pages'><refentrytitle>environ</refentrytitle><manvolnum>7</manvolnum></citerefentry> for details
-        about environment variables.</para></listitem>
+        <para>See "Environment variables in spawned processes" below for a description of how those
```

Comment:
usually we uppercase most first characters of words in section titles, no?

Reply:
For some reason, the target title wasn't uppercased (along with a few others). All fixed now.